internal/log/log.go

```diff
@@ -0,0 +1,37 @@
+package log
+
+import (
+	"fmt"
+	"github.com/logrusorgru/aurora"
+)
+
+// Au Aurora instance used for colors
+var Au aurora.Aurora
+
+// Print print a message with formatting
+func Print(part string, parts ...interface{}) {
+	hoverPrint()
+	fmt.Println(fmt.Sprintf(part, parts...))
+}
+
+// Fatal print a error with formatting (red)
+func Fatal(part string, parts ...interface{}) {
+	hoverPrint()
+	fmt.Println(Au.Colorize(fmt.Sprintf(fmt.Sprintf("%v", part), parts...), aurora.RedFg).String())
+}
+
+// Warn print a warning with formatting (yellow)
+func Warn(part string, parts ...interface{}) {
+	hoverPrint()
+	fmt.Println(Au.Colorize(fmt.Sprintf(fmt.Sprintf("%v", part), parts...), aurora.YellowFg).String())
+}
+
+// Info print a information with formatting (green)
+func Info(part string, parts ...interface{}) {
```

Comment:
`Print`, `Fatal` (`Error`), `Warn` and `Info` all accept formatting strings. For readability it may make sense to name them `Printf`, `Errorf`, `Warnf`, `Infof`. This matches expectations of most Go developers.